src/components/views/MonitoringChart.js

```diff
@@ -15,7 +17,16 @@ export default props => {
                 layout={{
                     width: props.fullWidth ? 1200 : 600,
                     height: props.fullWidth ? 350 : 250,
-                    yaxis: {showline: true},
+                    yaxis: {
+                        showline: true,
+                        title: 'Clients',
+                        titlefont: fontStyles,
+                        format: format(',d'),
```

Comment:
Do I understand correctly that this adds commas? If it does, we could do the same without adding a dependency by using this vanilla JS:

```
$ node
> const n = 500000;
undefined
> n.toLocaleString('en-US');
'500,000'
```

Reply:
It's decimal notation - rounded to integers. The purpose here is to disable ticks of fractional numbers when the y axis has low values (ie we don't have 1.5 clients that unenrolled). This approach might be heavy locally but the production bundle should have this anyway since plotly depends on D3.

I wouldn't be opposed to writing my own implementation though plotly expects a formatting function as the value here.

Reply:
I wonder if it would be possible to do something like:

```
format: d => Math.round(d),
```

just in case we switch from Plotly to something else not backed by d3 in the future.